HomeOxygenTherapy/R4/files/HomeOxygenTherapyPrepopulation-0.1.0.cql

```diff
@@ -9,39 +9,35 @@ include FHIRHelpers version '4.0.0' called FHIRHelpers
 // with guidance from https://www.cms.gov/Outreach-and-Education/Medicare-Learning-Network-MLN/MLNProducts/Downloads/Home-Oxygen-Therapy-ICN908804.pdf
```

Comment:
Overall, this file could use some better documentation to show how sections on grouped. For example, add comments to indicate where valuesets are defined, helper functions are defined, etc.